src/conf_functions/pick_action_functions.rs

```diff
@@ -0,0 +1,21 @@
+use crate::Action::{Combat, Reproduce};
```

Comment:
I really dislike this import. I believe it's an established convention to not import enum variants this way. I suggest removing this and quantifying `Action`s in this code.